src/trip/components/tripsTable.jsx

```diff
@@ -46,44 +49,48 @@ const TripsTable = ({ trips, users, isBack }) => {
-                                    onOpen={() => setDeletePopup(true)}
-                                    onClose={handleDeleteTrip}
-                                >
-                                    <Segment
-                                        style={{
-                                            left: '40%',
-                                            position: 'fixed',
-                                            top: '40%',
-                                            zIndex: 1000,
-                                        }}
-                                    >
-                                        <Header>Êtes vous sûr de vouloir supprimer ce trajet ?</Header>
-                                        <Button size='mini' content='Confirmer' negative onClick={handleDeleteTrip}/>
-                                    </Segment>
-                                </Portal>
+                                <ul>{
+                                    Array(trip.seats).fill().map((_, index) => (
+                                        <li key={`passenger-${tripIndex}-${index}`}>{trip.passengers[index] ? getUserById(trip.passengers[index]).username : 'Place libre'}</li>
+                                    ))
+                                }</ul>
                             </Table.Cell>
-                        : null }
-                    </Table.Row>
-                ))}
+                            { login ?
+                                <Table.Cell>
+                                    <TripsModal mode='edit' initialData={trip} disabled={login !== getUserById(trip.driver).username}>Modifier</TripsModal>
+                                    <Portal
+                                        openOnTriggerClick
+                                        open={deletePopup}
+                                        trigger={<Button size='mini' disabled={login !== getUserById(trip.driver).username}  content='Supprimer'/>}
+                                        onOpen={() => setDeletePopup(true)}
+                                    >
+                                        <Segment
+                                            style={{
+                                                "border": "solid 1px #666",
+                                                left: '40%',
+                                                position: 'fixed',
+                                                top: '40%',
+                                                zIndex: 1000,
+                                                textAlign: "center"
+                                            }}
+                                        >
+                                            <Header>Êtes vous sûr de vouloir supprimer ce trajet ?</Header>
+                                            <Button size='mini' content='Annuler' onClick={() => setDeletePopup(false)}/>
```

Comment:
Can't you just make a `setDeletePopup` that opens it when it's closed and closes it when it's open ? Not sure if it's easy with use state, but it's simpler to work with :)

Reply:
I dont know if i can / should override the useState setter behaviour (if it's what you are talking about), but i could create a separate function which doesn't take arguments and uses the setDeletePopup function.

Should i do that ?

Reply:
This thing about it is quite interesting https://dev.to/alexkhismatulin/update-boolean-state-right-with-react-hooks-3k2i, no need/interest to do it now we could do it later if we have several booleans. Just add an issue :)

Reply:
Ok, i will add another issue.

Reply:
Done in #75